.github/workflows/doc-build.yml

```diff
@@ -27,6 +27,9 @@ on:
 env:
   # NOTE: west docstrings will be extracted from the version listed here
   WEST_VERSION: 0.11.1a1
+  # The latest CMAke available directly with apt is 3.18, but we need >=3.20
```

Comment:
nit: `CMAke` -> `CMake` (if you make any other changes to this PR.)

Reply:
Fixed